src/lib/_imports/elements/html-elements.cms.css

```diff
@@ -57,8 +57,6 @@ Styleguide Elements.TextContent
 & :where(
   ol, ul,
   dl dl /* Treat nested definition lists like other nested lists */
-):not(
-  :--form *
 ) {
   padding-left: var(--global-space--list-indent);
 }
```

Comment:
**Why?**
To avoid taking effect instead of CSS that is loaded later with a less specific selector.